content/docs/introduction/first_steps.md

```diff
@@ -86,24 +86,24 @@ tab.
 
 As you can gather from http://localhost:9090/metrics, one metric that
 Prometheus exports about itself is called
-`http_requests_total` (the total number of HTTP requests the Prometheus server has made). Go ahead and enter this into the expression console:
+`promhttp_metric_handler_requests_total` (the total number of `/metrics` requests the Prometheus server has made). Go ahead and enter this into the expression console:
```

Comment:
served, not made

Reply:
You might want prometheus_http_request_duration_seconds_count instead

Reply:
@brian-brazil Updated. Thanks.